lib/cretonne/meta/isa/riscv/encodings.py

```diff
@@ -153,3 +156,75 @@
 RV32.enc(base.regmove.i32, Irmov, OPIMM(0b000))
 RV64.enc(base.regmove.i64, Irmov, OPIMM(0b000))
 RV64.enc(base.regmove.i32, Irmov, OPIMM32(0b000))
+
+
+def rv32_enc_c(inst, recipe, bits):
+    RV32.enc(inst, recipe, bits, isap=use_c)
```

Comment:
Note that encoding recipes can have their own `instp` and `isap` predicates that are automatically combined with the per-opcode predicates. That is probably easier to use that this function.

Reply:
Ah thanks, I'll get that in the next commit